### ISSUE TITLE
server class example

### ISSUE THREAD

**Issue by cretace** [leafo/scssphp#516]
Could you please rewrite your Example page on "Server" to make it clear that the Server class is NOT added via the standard 'scss.inc.php'. 
I have no idea why you moved it but a few notes on that page might stop someone else spending 5 hours scratching their head why it is not working!

**Comment by celemack** [comment]
Yes please -- this information would have been really helpful to me also!
